core/src/main/scala/codesearch/core/index/LanguageIndex.scala

```diff
@@ -134,5 +137,9 @@ trait LanguageIndex[VTable <: DefaultTable] { self: DefaultDB[VTable] =>
     * @param version of package
     * @return count of downloaded files (source files)
     */
-  protected def updateSources(name: String, version: String): Future[Int]
+  protected def updateSources(name: String, version: String): IO[Int]
+}
+
+object LanguageIndex {
+  val ConcurrentTasksCount = 30
```

Comment:
Need to choose the correct number.

Reply:
It might be different for different repositories, though. Anyway, I can try it out on the server.

Reply:
Can it be in the config, though? Or would that be inconvenient?